tests/AdaptiveExpressions.Tests/ExpressionParserTests.cs

```diff
@@ -1014,19 +1012,19 @@ public class ExpressionParserTests
            || value is double
            || value is decimal;
 
-        [DataTestMethod]
-        [DynamicData(nameof(Data))]
+        [Theory]
+        [MemberData(nameof(Data))]
         public void Evaluate(string input, object expected, HashSet<string> expectedRefs)
         {
             var parsed = Expression.Parse(input);
-            Assert.IsNotNull(parsed);
+            Assert.NotNull(parsed);
             var (actual, msg) = parsed.TryEvaluate(scope);
-            Assert.AreEqual(null, msg);
+            Assert.Null(msg);
             AssertObjectEquals(expected, actual);
             if (expectedRefs != null)
             {
                 var actualRefs = parsed.References();
-                Assert.IsTrue(expectedRefs.SetEquals(actualRefs), $"References do not match, expected: {string.Join(',', expectedRefs)} acutal: {string.Join(',', actualRefs)}");
```

Comment:
>$"References do not match, expected: {string.Join(',', expectedRefs)} acutal: {string.Join(',', actualRefs)}"); [](start = 66, length = 111)

Is it OK to drop error messages in this and other similar tests?

Reply:
Xunit does not allow custom error messages. If they still exist in other places (that's not in this PR) they need to be removed.